platform/android/java/app/build.gradle

```diff
@@ -20,7 +20,6 @@ allprojects {
     repositories {
         mavenCentral()
         google()
```

Comment:
Should we make the repo order consistent everywhere? Most seem to be Google then Maven, aside from this one.

Reply:
Done!